README.md

```diff
@@ -8,11 +8,12 @@ Running on a desktop requires [docker](https://www.docker.com/get-started).
 
 ## Installation
 
-First, adjust the environment setting in the [configuration](./hercules/settings/config.ini). On the Yale cluster you enter 'grace', while on a desktop (or laptop) you enter 'desktop' (both without quotes).
+First, adjust the environment setting in the [configuration](./hercules/settings/config.ini). On the Yale cluster you enter 'grace', while on a desktop (or laptop) you enter 'desktop' (both without quotes). For usage on a desktop you might also want to adjust the number of parallel jobs. In theory you can set it as high as the number of logical cores in your system. However, depending on your simulation settings Locust can consume a lot of RAM. Therefore, using as many cores as possible can potentially overload the RAM. In that case your system will become unusable. Do not touch the rest of the file.
 ```
 [USER]
 #possible values are 'desktop', 'grace'
 ENVIRONMENT = grace
+PARALLEL_JOBS = 2
```

Comment:
Would it be possible to just change this to?
```suggestion
DESKTOP_PARALLEL_JOBS = 2
```
Just for clarity.

Reply:
You are right about this one. I will change it.